.github/workflows/test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         os: [macos-latest, ubuntu-latest]
     steps:
       - uses: actions/checkout@v2
-      - uses: fwal/setup-swift@v1
+      - uses: swift-actions/setup-swift@v1
```

Comment:
Ugh, this is annoying it doesn't support concurrency. 

I spent a little time trying to figure out how Vapor is successfully testing their async code on macos, and found the `macos-unit` line of [this file](https://github.com/vapor/ci/pull/5/files#diff-c772b0463bfd2af4852bdf84778c809b5147e6a3ec217d05ad28bb4c88d12f16). It looks like instead of using `swift-setup`, they instead use `runs-on: macos` with `maxim-lobanov/setup-xcode@v1`. 

Do you mind either playing around with that solution, or just commenting out the macos tests with a link to [the  swift-setup issue you referenced](https://github.com/swift-actions/setup-swift/issues/415)?

Reply:
I'll look into it and start working on it probably later on the day.

Reply:
Alright, I think I got the setup-xcode workflows working

Reply:
Oh awesome! Thanks for doing that.